ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/StandaloneExecutor.java

```diff
@@ -65,17 +68,21 @@
   private final CountDownLatch shutdownLatch = new CountDownLatch(1);
   private final Map<String, Object> configProperties = new HashMap<>();
   private final boolean failOnNoQueries;
+  private final VersionCheckerAgent versionCheckerAgent;
```

Comment:
Out of curiosity, why is it that we previously didn't collect metrics via KsqlVersionChecker in standalone mode?